readme.md

```diff
@@ -132,7 +132,7 @@ server {
     ssl_early_data on;
 
     # Add Alt-Svc header to negotiate HTTP/3.
-    add_header alt-svc 'h3-27=":443"; ma=86400, h3-28=":443"; ma=86400, h3-29=":443"; ma=86400';
+    add_header alt-svc 'h3=":443"; ma=86400';
```

Comment:
Interestingly, traefik (when running with an experimental http3 support) does emit this header:

```
alt-svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
```

Reply:
Probably the server that they are using supports h3-29 draft version but the latest nginx removed all draft versions remember?